README.md

```diff
@@ -1,6 +1,9 @@
 scamandrill
 ===========
 
+[![Build Status](https://travis-ci.org/scamandrill/scamandrill.svg?branch=master)](https://travis-ci.org/scamandrill/scamandrill)
```

Comment:
Can you use https://img.shields.io/travis/USER/REPO.svg etc for consistent badge appearance?